### ISSUE TITLE
Mockup for outreach page

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#718]
[Mockup](https://gwu0.sharepoint.com/:p:/r/sites/GlyGenTeam-GRP/_layouts/15/Doc.aspx?sourcedoc=%7B57FF9715-7E54-4C57-A8A2-A0A919FDD145%7D&file=2023.09.27-Outreach%20page.pptx&action=edit&mobileredirect=true)

**Blocker for:**
* #768 
* #769

**Comment by ReneRanzinger** [comment]
Add JSON templates for Robel.

**Comment by ReneRanzinger** [comment]
@rykahsay I added an example JSON file for the webservices to maintain outreach. 

[input.json](https://github.com/glygener/glygen-issues/files/13329049/input.json)

I suggest to use for all web service (insert, get, update) the exact same JSON structure. It has the implication that with insert and update multiple entries can be added or modified at once, but it also allows us to have a single JSON and even us this JSON as dump to replicate entries from production to beta and tst. For each type of activity I added one array in the JSON. That’s a personal preference. I think (except for paper) we could have them all in one array with a type field but that would make the mandatory/optional configuration more complicated.

Some comments: 
* ID field cannot be there for insert but is mandatory for get and update. I leave it to you if you want to make one global autoincrement for IDs or have one for each array as shown in the example.
* All dates follow the date formatting of ISO 8601.
* For “papers” all fields are required except for issue and pages. For pmid and doid, one of them or both must be there.
* For “talks”, “posters”, “demos”, “virtual_talks”, “workshops” and “training” ‘files’ is the only optional field.

Let me know if you have questions.
